pkg/spec/gpyrpc/gpyrpc.proto

```diff
@@ -263,8 +266,8 @@ message ValidateCode_Result {
 }
 
 message CodeSnippet {
-    string schema = 1;
-    string rule = 2;
+	string schema = 1;
+	string rule = 2;
```

Comment:
Avoid unnecessary diffs.

Reply:
these diffs are intended to keep consistent code format